src/queries/pet/updatePetQuery.ts

```diff
@@ -0,0 +1,17 @@
+import { type IPet } from '../../interfaces/fakeDataTypes'
+import { Pet } from '../../models'
+
+const updatePetQuery = async (postId: number, petData: IPet): Promise<IPet | null> => {
+  const [count, [updatedPet]] = await Pet.update(
+    { ...petData },
+    {
+      where: { postId },
+      returning: true
+    }
+  )
+  if (count === 0) {
+    return null
+  }
```

Comment:
I believe this should be in the controller not in the query since the query should only do the query and returning results not a response and shape of data